src/views/Users.vue

```diff
@@ -1,19 +1,41 @@
 <template>
-  <PageViewer :pageTitle="pageTitle" :content="content" />
+  <user-list :users="users" />
 </template>
 <script>
 import PageViewer from "../components/PageViewer.vue";
```

Comment:
We are longer using the PageViewer. Please, remove it

Reply:
I removed it now

Reply:
It's not imported you can see it in the code that only the UserItem is imported